cms/envs/common.py

```diff
@@ -41,12 +41,8 @@
 from xmodule.modulestore.edit_info import EditInfoMixin
 
 ############################ FEATURE CONFIGURATION #############################
+STUDIO_NAME = "Studio"
```

Comment:
Also, I wonder if you should support i18n on the name. I know that Studio doesn't get translated, but I could imagine that some Open edX installations might want to have a different name for different languages, e.g. a Chinese name and an English version.